src/include.js

```diff
@@ -17,8 +18,12 @@ const preload = (content, resourcePath) => {
 
   const root = shared.resolveAssetsPath('includes');
 
-  includes.each((i, elem) => {
+  includes.each((idx, elem) => {
     const $elem = $(elem);
+    $elem.attr('fileName', (idx, attrValue) => {
+      return stacheJsonDataService.replaceWithStacheData(attrValue);
```

Comment:
The method name `replaceWithStacheData` doesn't really tell me what's going to happen when I call it. Something like `parseAngularBindings` is a bit more descriptive ('WithStacheData' is redundant as the method is part of the JSON data utility); maybe something along those same lines?